packages/components/utils/src/Portal/Portal.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Portal'
+type: 'component'
+status: 'beta'
+slug: /utils/portal/
+typescript: ./Portal.tsx
+---
+
+import { Props } from '@contentful/f36-docs-utils';
+
+`Portal` component can be used to render children in to a DOM node outside of the parent component. It can be useful for rendering components, you dont want to be hidden, clipped by parent component. For example, models, popovers, dropdowns.
```

Comment:
```suggestion
`Portal` component can be used to render children in a DOM node outside of its parent component. It can be useful for rendering components that you don’t want to be clipped by their parent’s component. For example: modals, popovers, dropdown, etc.
```